### PR TITLE
Reduce the time spend in StringMemoryTest

### PR DESCRIPTION
**Pull request by aschwaighofer**
Measure the increase.

rdar://problem/48658000

### CODE REVIEW

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Comment by aschwaighofer** [comment]
#23146

File: validation-test/stdlib/StringMemoryTest.swift [hunk 1]
```diff
@@ -12,6 +12,14 @@ import Foundation
 let str = "abcdefg\u{A758}hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz\u{A759}"
 let str2 = "abcdefg\u{A759}hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz\u{A758}"
 
+
+@inline(never)
+func getMemoryUsage() -> Int {
+  var usage = rusage()
+  getrusage(RUSAGE_SELF, &usage)
```
Comment (compnerd) [inline validation-test/stdlib/StringMemoryTest.swift]: This wont work on Windows, and should use `GetProcessMemoryInfo` there.

Comment (aschwaighofer) [reply validation-test/stdlib/StringMemoryTest.swift]: This is not a new test - the PR just moves this code within the test.

I think this test is disabled on windows by requiring: ``// REQUIRES: objc_interop  ``

**Pull request merged**